geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HExistsExecutor.java

```diff
@@ -55,11 +55,6 @@ public void executeCommand(Command command, ExecutionHandlerContext context) {
     ByteArrayWrapper key = command.getKey();
     try (AutoCloseableLock regionLock = withRegionLock(context, key)) {
       RedisHash map = getMap(context, key);
-
-      if (map == null || map.isEmpty()) {
-        command.setResponse(Coder.getIntegerResponse(context.getByteBufAllocator(), NOT_EXISTS));
```

Comment:
can NOT_EXISTS be removed?

Reply:
`NOT_EXISTS` is still used a few lines down, so I left it.